gradio/analytics.py

```diff
@@ -27,15 +27,14 @@ def analytics_enabled() -> bool:
 
 
 def _do_analytics_request(url: str, data: dict[str, Any]) -> None:
+    data["ip_address"] = get_local_ip_address()
     try:
         requests.post(url, data=data, timeout=5)
     except (requests.ConnectionError, requests.exceptions.ReadTimeout):
         pass  # do not push analytics if no network
 
 
 def version_check():
-    if not analytics_enabled():
```

Comment:
This API is broken - we're returning version 3.14 as the latest version lol

Reply:
We haven't updated this since it prints an annoying message to anyone using an older version. Specifically chose 3.14 as anyone using an older version uses an insecure way to generate share links

But if needed this, can be updated here; https://github.com/gradio-app/gradio-api-server/blob/1b62edbd86eed452d901f4f0b76bb7824ca45827/server.py#L31

Reply:
Got it. Yea would be annoying to receive every time you run an app. If it’s annoying maybe we can remove this function entirely?

Reply:
I think its still valuable in rare cases we have a security situation and we want to push a message out to all users telling them to upgrade

Reply:
Sound good